SubmittingPatches

```diff
@@ -260,17 +259,20 @@ A: The target branch depends on the nature of your change:
    If you are adding a feature, target the "master" branch in your pull
    request.
 
-   If you are fixing a bug, target the "next" branch in your pull request.
-   The Ceph core developers will periodically merge "next" into "master". When
-   this happens, the master branch will contain your fix as well.
+   If you are fixing a bug, target the named branch corresponding to the
+   major version that is currently in development. For example, if
+   Infernalis is the latest stable release and Jewel is development, target
+   the "jewel" branch for bugfixes. The Ceph core developers will
+   periodically merge this named branch into "master". When this happens,
```

Comment:
merge this named branch into "master" after the next development release is published (approximately once a month).